.circleci/config.yml

```diff
@@ -188,17 +192,15 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            # when lock file changes, use increasingly general patterns to restore cache
-            # If this gets our of sync, you can increment the version (vX).  Just be
-            # sure to match that version when saving the cache as well.
-            - node-v1-{{ checksum "package-lock.json" }}
-            - node-v1-
+            - <<pipeline.parameters.cache-version>>-{{ checksum "package-lock.json" }}-{{ checksum "prisma/schema.prisma" }}
+            - <<pipeline.parameters.cache-version>>-{{ checksum "package-lock.json" }}
+            - <<pipeline.parameters.cache-version>>-
```

Comment:
Most of this looks good, but I did want to ask just to be positive, do we really want these more general cache keys?

I feel like this can potentially suppress errors here until further in the deployment process. The `db:generate-client` script doesn't do any cleanup, so it only makes additive changes to `node_modules/.prisma/client`.  If we ever remove a `binaryTarget`, or if prisma changes its file structure, these more generic cache keys will prevent those files from ever being cleaned up.  We could also change the `db:generate-client` script to remove the `node_modules/.prisma` directory as well to throw out another option.

Reply:
I see no reason to keep the more general keys around, this is mostly just copy-pasta from my own testing. What you're saying makes sense and I can update this PR to match.